cpe/openscap-cpe-dict.xml

```diff
@@ -17,6 +17,18 @@
             <title xml:lang="en-us">Red Hat Enterprise Linux 7</title>
             <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.rhel:def:7</check>
       </cpe-item>
+      <cpe-item name="cpe:/o:oracle:linux:5">
+            <title xml:lang="en-us">Oracle Linux 5</title>
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.ol:def:5</check>
+      </cpe-item>
+      <cpe-item name="cpe:/o:oracle:linux:6">
+            <title xml:lang="en-us">Oracle Linux 6</title>
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.ol:def:6</check>
+      </cpe-item>
+      <cpe-item name="cpe:/o:oracle:linux:7">
```

Comment:
Looks like a typo in docs, I'll contact tech writers about that. Please find officially approved CPE names:
https://nvd.nist.gov/products/cpe/search/results?keyword=cpe%3a%2fo%3aoracle%3alinux&status=FINAL&orderBy=CPEURI&namingFormat=2.2
Correct CPE names expected to be:
cpe:/o:oracle:linux:5
cpe:/o:oracle:linux:6
cpe:/o:oracle:linux:7